### PR TITLE
SDK -303: Optimising Artifact.java to reduce clutter

### PR DESCRIPTION
**Pull request by prateek27das**
Optimising Artifact.java to reduce clutter

### CODE REVIEW

File: sdk-commons/src/main/java/org/openmrs/maven/plugins/model/Artifact.java [hunk 1]
```diff
@@ -177,7 +180,9 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return groupId + ':' + artifactId + ':' + version;
+        StringBuilder sb = new StringBuilder();
```
Comment (ibacher) [inline sdk-commons/src/main/java/org/openmrs/maven/plugins/model/Artifact.java:183]: Don't use a `StringBuilder` the string concatenation here is actually the right concatenation technique to use.

**Review by ibacher — CHANGES_REQUESTED** [review]
Thanks for this!

File: README.md [hunk 2]
```diff
@@ -6,8 +6,22 @@ For more details visit [the project page](https://wiki.openmrs.org/display/docs/
 
 ## Requirements
  * Maven 3.x
+
```
Comment (ibacher) [inline README.md:9]: It's probably better to do several branches off of master rather than branching off as-yet unmerged PRs.

**Comment by brandones** [comment]
Closing as stale. @prateek27das please feel free to re-open if you continue working on it.

**Pull request closed without merging**